tools/update-pdfjs

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# This script fetches the latest build of PDF.js from the viewer demo
+# page.
+#
+# See https://github.com/mozilla/pdf.js/wiki/Setup-pdf.js-in-a-website#from-examples
+
+DEST_DIR=src/vendor/pdfjs
+PREFIX=pdf.js-gh-pages
+COMPONENTS="$PREFIX/build $PREFIX/web $PREFIX/LICENSE"
+
+rm -rf $DEST_DIR
+mkdir -p $DEST_DIR
+
+# Get the latest build of the viewer
+curl -L https://github.com/mozilla/pdf.js/archive/gh-pages.tar.gz \
+  | tar -xz --directory $DEST_DIR --strip-components=1 $COMPONENTS
+
+# Remove example content from viewer
+rm $DEST_DIR/web/*.pdf
+
+# Remove the check that the PDF being loaded is from the same origin as the
+# viewer
+sed -i '/^\s*validateFileURL(file)/d' $DEST_DIR/web/viewer.js
+
+# Add code to load the Hypothesis client when the viewer runs
+sed -i '/<head>/a <script async defer src="\/public\/embed.js"></script>' $DEST_DIR/web/viewer.html
```

Comment:
Oops, I suppose my comment about the `defer` attribute should probably have gone here...